### ISSUE TITLE
DateTime relative formatter shows "in 0 seconds" instead of "0 seconds ago"

### ISSUE THREAD

**Issue by eproxus** [bitwalker/timex#185]
### Steps to reproduce

```elixir
iex(22)> Timex.Format.DateTime.Formatters.Relative.format!(Timex.DateTime.now, "{relative}")
"in 0 seconds"
```

### Description of issue

The relative datetime formatters returns `"in 0 seconds"` instead of `"0 seconds ago"` when the time is now.

**Comment by eproxus** [comment]
Maybe it should even be `"now"` when the time is around 0 seconds.

**Comment by bitwalker** [comment]
Yeah it should be "now", I'll get that fixed

**Comment by bitwalker** [comment]
Fixed in 3.0

**Comment by eproxus** [comment]
Thanks for the fix, works brilliantly!
